cmd/main.go

```diff
@@ -123,6 +127,22 @@ func main() {
 		klog.Fatalf("Failed to initialize cloud provider: %v", err)
 	}
 
+	var kubeClient *kubernetes.Clientset
+	if *featureMaxSharePerInstance && *runController && *enableMultishare {
+		clusterConfig, err := buildConfig(*kubeconfig)
```

Comment:
a heads up that in #465 I've moved the `buildConfig()` function to `gcfs_driver.go` to avoid passing kubeclient around. let's decide where should it stay

Reply:
Let's address in that PR or a separate PR